msg/tunnel.msg

```diff
@@ -0,0 +1,6 @@
+uint64 timestamp	# time since system start (microseconds)
+uint16 payload_type  # A code that identifies the content of the payload (0 for unknown, which is the default). If this code is less than 32768, it is a 'registered' payload type and the corresponding code should be added to the MAV_TUNNEL_PAYLOAD_TYPE enum. Software creators can register blocks of types as needed. Codes greater than 32767 are considered local experiments and should not be checked in to any widely distributed codebase.*/
```

Comment:
Add the enums as constants in here. We want to technically keep Mavlink completely separate from uORB. 

https://mavlink.io/en/messages/common.html#MAV_TUNNEL_PAYLOAD_TYPE